src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/dashboard/fix/high-risk-data-breaches/HighRiskBreachLayout.tsx

```diff
@@ -120,6 +120,30 @@ export function HighRiskBreachLayout(props: HighRiskBreachLayoutProps) {
       setIsResolving(false);
     }
   };
+
+  let markedAsFixedInstrumentation;
+
+  switch (props.type) {
+    case "social-security-number":
+      markedAsFixedInstrumentation = "marked_fixed_social_security";
+      break;
```

Comment:
Here's a fun trick, which would also ensure we get a warning if we ever add something to `HighRiskBreachTypes` without adding an event string here, I think:

```ts
const eventMap: Record<HighRiskBeachTypes, string> = {
  "social-security-number": "marked_fixed_social_security",
  "credit-card": "marked_fixed_credit_card",
  // ..etc
};
const markedAsFixedInstrumentation = eventMap[props.type];
```